.gitignore

```diff
@@ -6,3 +6,4 @@
 venv/
 .pytest_cache/
 **__pycache__/
+.venv
```

Comment:
There is an extra space after `.venv`. Remove the space to ensure `.venv` directories are correctly ignored.